### ISSUE TITLE
Hide the firefox browser or virtual browser for window users..!

### ISSUE THREAD

**Issue by ankitjain28may** [ankitjain28may/FbPostUsingPython#1]
Selenium opens Firefox browser for executing the commands given to it.. But I want that the browser will not open or it may open.. But it should be virtual so user would not see it..!

**Comment by ankode** [comment]
i recommend using "mechanize" module.

**Comment by ankitjain28may** [comment]
Would u like to work on it @ankode

**Comment by suryatejreddy** [comment]
you can use pyvirtualdisplay, it will work for both windows and linux

**Comment by ankitjain28may** [comment]
Would u like to work on it

**Comment by suryatejreddy** [comment]
Yeah, the commit which i made before will work.

**Comment by ankitjain28may** [comment]
Send pr on contribution branch

**Comment by suryatejreddy** [comment]
install pip on windows (http://arunrocks.com/guide-to-install-python-or-pip-on-windows/)
 then from terminal type , pip install xvfbwrapper

**Comment by ankitjain28may** [comment]
If i have made this script so obviously i hve pip installed and i hve checked it after installing xvfbwrapper but still its not working try to check it on your system(Wndows)

**Comment by dibyasonu** [comment]
Is this issue still open?? I hope to work on it if it is still open

**Comment by ankitjain28may** [comment]
Yes, this issue is still open.. You can work on this issue, you can use phantomjs browser

**Comment by dibyasonu** [comment]
Its not working with phantomjs , the login.click() command is not working after using phantomjs. Should i send a PR, so that u can tell what's the problem and i can try to fix.

**Comment by ankitjain28may** [comment]
No, I will clone your repo and check it out.

**Comment by dibyasonu** [comment]
k..let me upload it..

**Comment by ankitjain28may** [comment]
Sure

**Comment by dibyasonu** [comment]
Done...created it under  a branch as it requires further tweaks
[fb_using_python](https://github.com/dibyasonu/FbPostUsingPython/tree/dibyasonu-patch-1)

Please do tell me how to fix it. I am a beginner in python as well as opensource contribution and hope to learn as much as possible. I am partcipating in the hacktoberfest too, so hoping to make a succesful pull request to this project

**Comment by ankitjain28may** [comment]
@dibyasonu Are you able to login using firefox ?

**Comment by ankitjain28may** [comment]
login.click() isn't working because facebook has changed the id of the login button
```
login = driver.find_element_by_css_selector("#u_0_5")
    login.click()
```
Try with this code or you can get it using developer tools of the browser.

**Comment by dibyasonu** [comment]
i have **fixed the fb.py and sent a PR**. I have mentioned about the changes in the PR comments, please tell me if something else is needed.  
And i think the **README.md needs to be changed** too, as phantomjs browser is now being used in the project. So if u want i can create a PR for it. Please reply soon.

**Comment by ankitjain28may** [comment]
Ya sure make a new PR for readme.. N i will check changes soon..

**Comment by dibyasonu** [comment]
Did it work....?? I have already sent the PR
